Aliases.php

```diff
@@ -21,3 +21,23 @@ class LatLongValue extends \DataValues\Geo\Values\LatLongValue {}
 	class GlobeCoordinateValue extends \DataValues\Geo\Values\GlobeCoordinateValue {}
 
 }
+
+namespace DataValues\Geo\Formatters {
+
+	/**
+	 * @since 1.0
+	 * @deprecated since 2.0, use the base class instead.
+	 */
+	class GeoCoordinateFormatter extends \DataValues\Geo\Formatters\LatLongFormatter {}
+
+}
+
+namespace DataValues\Geo\Parsers {
+
+	/**
+	 * @since 1.0
+	 * @deprecated since 2.0, use the base class instead.
+	 */
+	class GeoCoordinateParser extends \DataValues\Geo\Parsers\LatLongParser {}
+
+}
```

Comment:
As I understood this is only for IDE. 
If so it is not needed, as soon as PhpStorm supports `class_alias` nowadays. See https://blog.jetbrains.com/phpstorm/2016/11/phpstorm-2016-3-rc1/